### DISCUSSION TITLE
How to center text in table headers?

### DISCUSSION THREAD

**Discussion by anunknowperson** [mantinedev/mantine#3018] [Q&A] [answered]
How to center text in https://mantine.dev/core/table/ headers?

my textAlign on th is being overridden by some other style.

![image](https://user-images.githubusercontent.com/54563399/202894517-3feac82c-3ce7-4d70-b996-0e5636f910e3.png)

**Comment by rtivital** [comment, marked as answer]
`<Table sx={{ '& thead tr th': { textAlign: 'center' } }} />`

**Reply by anunknowperson** [reply]
Thanks!
